### DISCUSSION TITLE
是否更换 论坛？

### DISCUSSION THREAD

**Discussion by kunpeng9** [kunpeng9/hulunote2020-11-12#22] [Q&A]
[葫芦笔记 - 乌托邦](https://web.uvw.org.cn/circle/s51A4n5) 这个的体验不太行，感觉是为移动端准备的，小程序用一下还行。
网页使用的话体验实在是不行。尤其往下拉的时候内容半天刷新不出来，有点慢。
要不干算在github搞算了。

**Comment by pimgeek** [comment]
这个确实不行，会换到 Discourse 论坛
